src/main/java/com/faforever/client/replay/ReplayCardController.java

```diff
@@ -85,7 +85,7 @@ public void setReplay(Replay replay) {
     timeLabel.setText(timeService.asShortTime(replay.getStartTime()));
     modLabel.setText(replay.getFeaturedMod().getDisplayName());
     playerCountLabel.setText(i18n.number(replay.getTeams().values().stream().mapToInt(List::size).sum()));
-    qualityLabel.setText(i18n.number((int) ((ratingService.calculateQuality(replay) * 10) / 10)));
+    qualityLabel.setText(i18n.number((int) ((ratingService.calculateQuality(replay) * 10) / 10)) + " %");
```

Comment:
Instead of hard-coding `+ " %"` we use i18n which allows using different symbols and also different text directions (right-to-left).

There's already a key `percentage` (IIRC) that can be used.